### PR TITLE
Display filepath of the failed behat scenarios

### PR DESCRIPTION
**Pull request by ahocquard**
[//]: <> (<3 Thanks for taking the time to contribute! You're awesome! <3)

[//]: <> (If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/.github/CONTRIBUTING.md)

**Description (for Contributor and Core Developer)**

[//]: <> (What does this Pull Request do? reference the related issue?)

**Definition Of Done (for Core Developer only)**

| Q                                 | A
| --------------------------------- | ---
| Added Specs                       | Todo
| Added Behats                      | Todo
| Added integration tests           | Todo
| Changelog updated                 | Todo
| Review and 2 GTM                  | Todo
| Micro Demo to the PO (Story only) | Todo
| Migration script                  | -
| Tech Doc                          | -

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

**Comment by ClementGautier** [comment]
Can't we override the Scenario class ? Would be much easier (we would need to override only the getTitle function)

File: features/category/display_category_history.feature [hunk 1]
```diff
@@ -18,7 +18,7 @@ Feature: Display the category history
     And I should see history:
       | version | property    | value         |
       | 1       | code        | book          |
-      | 1       | label-en_US | Book category |
+      | 1       | label-en_US | Book egor     |
```
Comment (grena) [inline features/category/display_category_history.feature]: I guess it was for debug purpose, but, all .feature files are not to merge isn't it?

File: features/Pim/Behat/Formatter/JUnitFormatter.php [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                $elemAttributes
+            );
+            $exception = str_replace(array('<![CDATA[', ']]>'), '', (string) $exception);
+            $xml .= sprintf(
+                "<![CDATA[\n%s\n]]></%s>\n",
+                $exception,
+                $elemType
+            );
+        }
+        $this->exceptions = array();
```
Comment (grena) [inline features/Pim/Behat/Formatter/JUnitFormatter.php]: Please replace all you `array()` to `[]`

File: features/Pim/Behat/Formatter/JUnitFormatter.php [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+            ? ', Ex #' . ($event->getIteration() + 1)
+            : '';
+        $caseStats = sprintf('classname="%s" name="%s" time="%F" assertions="%d"',
+            htmlspecialchars($className),
+            htmlspecialchars($name),
+            $time,
+            $this->scenarioStepsCount
+        );
+
+        $xml  = "    <testcase $caseStats>\n";
```
Comment (grena) [inline features/Pim/Behat/Formatter/JUnitFormatter.php]: extra space `$xml  =`

**Comment by ClementGautier** [comment]
As it will be part of our code I would say: don't create a xml by concatenate strings,  use a xml lib that can dump it properly.

**Comment by ahocquard** [comment]
@ClementGautier imho, it will be far better if we stick the much we can to the original junit formatter.
Reimplement a whole formatter to put 3 lines of code instead of 1 is overkilled.

**Comment by damien-carcel** [comment]
Also, be sure to use this new formatter on EE too.

**Pull request merged**